.github/workflows/ci.yml

```diff
@@ -158,8 +159,8 @@ jobs:
           sudo systemctl stop docker.service
           sudo apt-get purge docker-ce docker-ce-cli containerd.io docker-compose-plugin docker-ce-rootless-extras docker-buildx-plugin
           sudo apt-get install curl
-          curl -fsSL https://get.docker.com -o get-docker.sh
-          sudo sh ./get-docker.sh --version ${{ matrix.engine }}
+          curl -fsSL https://test.docker.com -o get-docker.sh
+          sudo REPO_FILE=docker-ce-staging.repo DOWNLOAD_URL=https://download-stage.docker.com sh ./get-docker.sh --version ${{ matrix.engine }}
```

Comment:
Using `download-stage.docker.com` as packages have not yet been deployed to production (will be done on Monday); I'll update the PR once that's done.